sdk/src/main/java/com/mapbox/maps/ResourcesAttributeParser.kt

```diff
@@ -20,29 +19,26 @@ internal object ResourcesAttributeParser {
     typedArray: TypedArray,
     credentialsManager: CredentialsManager
   ): ResourceOptions {
-    val accessTokenString: String =
-      typedArray.getString(R.styleable.mapbox_MapView_mapbox_resourcesAccessToken)
-        ?: credentialsManager.getAccessToken(context)
+    val builder = ResourceOptionsManager.getDefault(context).resourceOptions.toBuilder()
 
-    val cachePathString =
-      typedArray.getString(R.styleable.mapbox_MapView_mapbox_resourcesCachePath)
-        ?: "${context.filesDir.absolutePath}/$DATABASE_NAME"
+    if (typedArray.hasValue(R.styleable.mapbox_MapView_mapbox_resourcesAccessToken)) {
```

Comment:
Should apply `credentialsManager.getAccessToken(context)` in else block.

Reply:
The `val builder = ResourceOptionsManager.getDefault(context).resourceOptions.toBuilder()` should have already filled in the default access token, the xml configuration will only overwrite it when it's set.